fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -2635,10 +2635,19 @@ private String getRequestBaseUri(String type) throws Exception {
     /**
      * Builds a collection of properties that will be passed to the persistence interceptors.
      *
+     * @param type
+     *            the resource type
```

Comment:
super minor, but at some point we should agree on consistent formatting for our javadoc...we're pretty much all over with this at the moment

Reply:
Agreed. In this case I just went with the format being used on the other methods in the class.